demo/imports/lib/schema.js

```diff
@@ -21,17 +23,21 @@ MessageBox.defaults({messages: {en: {syntax: ''}}});
 
 // This is required for the eval.
 if (Meteor.isClient) {
-    window.GraphQLBridge  = GraphQLBridge;
-    window.SimpleSchema   = SimpleSchema;
-    window.SimpleSchema2  = SimpleSchema2;
-    window.buildASTSchema = buildASTSchema;
-    window.parse          = parse;
+    window.Ajv              = Ajv;
+    window.GraphQLBridge    = GraphQLBridge;
+    window.JSONSchemaBridge = JSONSchemaBridge;
+    window.SimpleSchema     = SimpleSchema;
+    window.SimpleSchema2    = SimpleSchema2;
+    window.buildASTSchema   = buildASTSchema;
+    window.parse            = parse;
 } else {
-    global.GraphQLBridge  = GraphQLBridge;
-    global.SimpleSchema   = SimpleSchema;
-    global.SimpleSchema2  = SimpleSchema2;
-    global.buildASTSchema = buildASTSchema;
-    global.parse          = parse;
+    global.Ajv              = Ajv;
+    global.GraphQLBridge    = GraphQLBridge;
+    global.JSONSchemaBridge = JSONSchemaBridge;
+    global.SimpleSchema     = SimpleSchema;
+    global.SimpleSchema2    = SimpleSchema2;
+    global.buildASTSchema   = buildASTSchema;
+    global.parse            = parse;
 }
```

Comment:
More and more of these... How about using `const scope = Meteor.isClient ? window : global;` and update this code accordingly?